app/main/controllers/activity/RTMediaBuddyPressActivity.php

```diff
@@ -281,7 +281,7 @@ function bp_after_activity_post_form() {
 			$url_upload                              = implode( '/', $slug_split );
 			$url                                     = trailingslashit( $url_upload ) . 'upload/';
 		} else {
-			$url = trailingslashit( $url ) . 'upload/';
+			$url = 'upload' . trim( $url, '/' );
```

Comment:
Why prefix upload in url ?
possible output `uploadhttps://yahil.rtdemo.in/?a=sds`